docs/content/workers/api-worker-system.md

```diff
@@ -12,7 +12,7 @@ The `WorkerSystem` class stores information about a worker during [Runtime]({{ur
 
 | Field             | Type                   | Description                    |
 |-------------------|------------------------|--------------------------------|
-| Connection    | [Connection]({{urlRoot}}/content/connecting-to-spatialos) | The connection to the SpatialOS Runtime. You can use it to send data and messages. |
+| Connection    | [`Connection`]({{urlRoot}}/content/connecting-to-spatialos) | The connection to the SpatialOS Runtime. You can use it to send data and messages. |
```

Comment:
```suggestion
| Connection    | [`Connection`]({{urlRoot}}/content/connecting-to-spatialos) | The connection to the SpatialOS [Runtime]({{urlRoot}}/content/glossary#spatialos-runtime). You can use it to send data and messages. |
```

Reply:
Soz, know that this is old text but backlinks to the glossary make everything OK.